beetsplug/acousticbrainz.py

```diff
@@ -142,7 +142,10 @@ def _get_data(self, mbid):
             self._log.debug("fetching URL: {}", url)
 
             try:
-                res = requests.get(url)
+                res = requests.get(
+                    url,
+                    timeout=10,
```

Comment:
✍🏼 A nitpick but try to keep this in a single line if it fits. Applies to the rest of the changes :)

Reply:
Done, except for couple of calls with 3+ args which become unreadable in a single line.